apps/web/src/server/views/applications/case-documentation/documentation-success-banner.njk

```diff
@@ -0,0 +1,39 @@
+{% extends "../layouts/applications.layout.njk" %}
+
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% set serviceName = "Document successfully deleted"  %}
+{% block pageTitle %}
+	{{ serviceName }} – {{ pageTitle | default('GOV.UK') }}
+{% endblock %}
+
+{% block beforeContent %}{% endblock %}
+{% block pageHeading %}{% endblock %}
+
+{% block pageContent %}
+	<div class="govuk-grid-row">
```

Comment:
do you need an H1 on this page?

Reply:
probably not. unless we want to change the gds html for the success banner